### PR TITLE
Add `loading: false` option to pages

### PR DESCRIPTION
**Pull request by galvez**
@Atinux finally got it to work, the example app should illustrate usage properly.

Feel free to edit if you think there's a better approach!

### CODE REVIEW

**Comment by galvez** [comment]
Thought I'd cc @ishitatsuyuki and @clarkdo on this too.

**Comment by manniL** [comment]
Could you add tests for the new feature? :)

**Comment by galvez** [comment]
@manniL yep! Working on it.

**Comment by galvez** [comment]
Ok, test is passing :)

**Comment by galvez** [comment]
@Atinux @manniL tests are passing!

**Comment by galvez** [comment]
Created [a PR in nuxt/docs][1] as well.

[1]: https://github.com/nuxt/docs/pull/677/files

File: lib/app/client.js [hunk 1]
```diff
@@ -115,7 +115,7 @@ async function loadAsyncComponents (to, from, next) {
   this._diffQuery = (this._queryChanged ? getQueryDiff(to.query, from.query) : [])
 
   <% if (loading) { %>
-  if (this._pathChanged && this.$loading.start) {
+  if (this._pathChanged && this.$loading.start && !this.$loading.manual) {
```
Comment (clarkdo) [inline lib/app/client.js]: This line should be called before `render` in `client.js`, but `manual` is initialized in `render`, so if in `spa` mode, can this line work ?

Comment (galvez) [reply lib/app/client.js]: @clarkdo good catch, I'll look into it.

Comment (galvez) [reply lib/app/client.js]: @Atinux @clarkdo so I didn't forget this, and I am happy to report the `loading: false` works flawlessly on SPA mode :)

**Pull request merged**